plugin/manager.go

```diff
@@ -173,9 +173,22 @@ func (pm *Manager) reload() error { // todo: restore
 		if validFullID.MatchString(v.Name()) {
 			p, err := pm.loadPlugin(v.Name())
 			if err != nil {
-				return err
+				if os.IsNotExist(errors.Cause(err)) {
```

Comment:
perhaps

```go
if os.IsNotExist(errors.Cause(err)) {
	// Likely some error while removing on an older version of docker
	logrus.WithError(err).WithField("id", filepath.Base(v.Name())).Warn("Missing plugin config, skipping. This may be caused due to a failed remove and requires manual cleanup.")
	continue
}
logrus.WithError(err).WithField("id", filepath.Base(v.Name())).Error("Error loading plugin, skipping.")
```